src/crypto/tls/handshake_server_tls13.go

```diff
@@ -718,6 +744,13 @@ func (hs *serverHandshakeStateTLS13) sendServerParameters() error {
 		return err
 	}
 
+	earlySecret := hs.earlySecret
```

Comment:
Out of curiosity, why move this code after the acceptance signal's computed?

Reply:
This change actually moves this chunk of code back to where it was before we implemented draft-09. (See `git diff go1.1.6 -- handshake_server_tls13.go`.)